cloud/deployment/deployment.go

```diff
@@ -749,139 +1102,295 @@ func Update(deploymentID, label, ws, description, deploymentName, dagDeploy, exe
 		}
 	}
 	// update deployment
-	d, err := client.UpdateDeployment(deploymentUpdate)
+	d, err := CoreUpdateDeployment(c.Organization, currentDeployment.Id, updateDeploymentRequest, platformCoreClient)
 	if err != nil {
 		return err
 	}
-
-	if d.ID == "" {
-		fmt.Printf("Something went wrong. Deployment %s was not updated", currentDeployment.Label)
+	if d.Id == "" {
+		fmt.Printf("Something went wrong. Deployment %s was not updated", currentDeployment.Name)
 	}
 
 	// do not print table if worker queue create or update was used
 	if !queueCreateUpdate {
 		tabDeployment := newTableOut()
 
-		runtimeVersionText := d.RuntimeRelease.Version + " (based on Airflow " + d.RuntimeRelease.AirflowVersion + ")"
-		clusterName := d.Cluster.Name
-		releaseName := d.ReleaseName
-		if organization.IsOrgHosted() {
-			if IsDeploymentHosted(d.Type) {
-				clusterName = d.Cluster.Region
-			}
-			releaseName = notApplicable
-		}
-		tabDeployment.AddRow([]string{d.Label, releaseName, clusterName, d.ID, runtimeVersionText, strconv.FormatBool(d.DagDeployEnabled), strconv.FormatBool(d.APIKeyOnlyDeployments)}, false)
+		runtimeVersionText := d.RuntimeVersion + " (based on Airflow " + d.AirflowVersion + ")"
+		releaseName := d.Namespace
+		clusterName := notApplicable
+		cloudProvider := *d.CloudProvider
+		region := notApplicable
```

Comment:
also here

Reply:
Fixing these when this code was first written you could not get cluster name from the request